srcpkgs/tree/template

```diff
@@ -1,17 +1,16 @@
 # Template file for 'tree'
 pkgname=tree
-version=2.0.0
+version=2.0.1
 revision=1
 build_style=gnu-makefile
 short_desc="Resursive directory listing program"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-or-later"
 homepage="http://mama.indstate.edu/users/ice/tree"
-#istfiles="http://deb.debian.org/debian/pool/main/t/tree/tree_${version}.orig.tar.gz"
-distfiles="http://mama.indstate.edu/users/ice/tree/src/tree-2.0.0.tgz"
-checksum=782cd73179f65cfca7f29326f1511306e49e9b11d5b861daa57e13fd7262889f
+changelog="http://mama.indstate.edu/users/ice/tree/changes.html"
+distfiles="http://mama.indstate.edu/users/ice/tree/src/tree-${version}.tgz"
```

Comment:
we switched to the debian mirror because indstate.edu is not accessible from everywhere: https://github.com/void-linux/void-packages/commit/fbcfe4eab8bdbdd5bb592c041722ab36d9718eda